core/arch/arm/include/mm/core_mmu.h

```diff
@@ -86,6 +86,7 @@
  * MEM_AREA_IO_SEC:   Secure HW mapped registers
  * MEM_AREA_RES_VASPACE: Reserved virtual memory space
  * MEM_AREA_TA_VASPACE: TA va space, only used with phys_to_virt()
+ * MEM_AREA_TEE_RAM_UNCACHED:  Same as MEM_AREA_TEE_RAM, but cache disabled.
```

Comment:
@etienne-lms Could you please share more info about memory identification test?  The only difference with TEE_RAM is cache disabled. I need that piece memory to be build into mmu with executable attribute set. COHERENT RAM does not reflect that attribute, I think.